eth/tracers/api_blockResult.go

```diff
@@ -0,0 +1,362 @@
+package tracers
```

Comment:
still think that it's not that common to have camelCase filename in golang.

Maybe we should rename to `eth/tracers/api_blockresult.go` & `eth/tracers/api_blockresult_test.go`

Reply:
changed.